.nvmrc

```diff
@@ -0,0 +1 @@
+v0.10.40
```

Comment:
I recommend switching to v0.10.46, which is what our Jenkins build jobs use, but LGTM otherwise.

Reply:
Oh I didn't know we could use 0.10.46, I just copied one from https://git.corp.ooyala.com/projects/PBW/repos/mjolnir/browse/.nvmrc. 
Are there any major differences between the two?

Reply:
Probably not, but I think it's better for our dev environment to use the same version as our build environment in order to avoid package version differences when doing npm install, etc.